doc/ref_arch/openstack/chapters/chapter03.md

```diff
@@ -104,15 +104,46 @@ Neutron is an OpenStack project that provides "network connectivity as a service
+-	Networking capabilities that can be instantiated, deployed, configured and managed like software. Network control is programmable and supports dynamic, manageable and adaptable networking.
+-	Support for both overlay and underlay networking
 
-- Decoupling of control and forwarding functions into control plane and data plane
-- Networking capabilities that can be instantiated, deployed, configured and managed like software. Network control is programmable and supports dynamic, manageable and adaptable networking.
-
-OpenStack Neutron supports open APIs and a pluggable backend where different plugins can be incorporated in the neutron-server. Plugins for various SDN controllers include the standard [ML-2 plugin](../chapter04.md#4234-neutron-ml2-integration) and vendor product specific [monolithic plugins](https://wiki.openstack.org/wiki/Neutron_Plugins_and_Drivers).
+OpenStack Neutron supports open APIs and a pluggable backend where different plugins can be incorporated in the neutron-server.
 
+Plugins for various SDN controllers include either the standard ML-2 plugin or specific monolithic plugins.
 Neutron supports both core plugins that deal with L2 connectivity and IP address management, and service plugins that support services such as L3 routing, Load Balancers, Firewalls, etc.
 
+Below we will explore an example of SDN controller under LFN projects , that can be used integrated with Neutron plugin which it help to overcome a lot of challenges in Vanilla Neutron and provide many features that can be consumed by VNF/CNF 
+
+
+#### 3.2.5.1. Tungsten Fabric (SDN Controller)
+Tungsten Fabric, an open source SDN in Linux Foundation Networking (https://tungsten.io/), offers neutron networking through ML2 based pugin, additionally it supports advanced networking features beyond basic neutron networking via monolithic plugin. It also supports the same advanced networking features via CNI plugin in Kubernetes. Hence, it works as a multi-stack SDN to support VMs, containers, and baremetal workloads. It provides separation of control plane functions and data plane functions with its two components:
+
+- Tungsten Fabric Controller– a set of software services that maintains a model of networks and network policies, typically running on several servers for high availability
+- Tungsten Fabric vRouter– installed in each host that runs workloads (virtual machines or containers), the vRouter performings packet forwarding and enforces network and security policies
+
+It is based on proven, standards-based networking technologies that today support the wide-area networks of the world’s major service providers, but repurposed to work with virtualized workloads and cloud automation in data centers that can range from large scale enterprise data centers to much smaller telco POPs. It provides many enhanced features over the native networking implementations of orchestrators, including:
```

Comment:
POPs acronym first use -- also is the term commonly used across the world?

Reply:
done